e2e/coverage-handlebars/transform-handlebars.js

```diff
@@ -5,16 +5,15 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+const {build, normalize} = require('@jridgewell/build-mapping');
```

Comment:
Sorry to keep changing things. I decided to make `build` normalize by default (it's actually a bit faster this way, avoids a tree recursion). There's a separate `unnormalizedBuild` function, but since you want normalized you should just stick with `build`.
```suggestion
const {build} = require('@jridgewell/build-mapping');
```

Reply:
nice!